samples/array-sample/README.md

```diff
@@ -0,0 +1,109 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Array input sample for Kubernetes Marketplace Offer
+
+This sample demonstrates how to override array inside values.yaml for Kubernetes Marketplace Offer
+
+In helm, there are 2 different ways to override array inside values.yaml
+- --set cpuLimit='{1,1.5}'
+- --set memoryLimit[0]=512Mi --set memoryLimit[1]=0.75Gi
+
+
+Both ways can be used in Kubernetes Marketplace Offer. However, the first way is preferred if you want to provide more flexibility to number of elements.
+
+## ARM Template
+
+The following code snipnet shows how the values are passed as configuration settings:
+
+```json
+"configurationSettings": {
+    "title": "[parameters('app-title')]",
+    "value1": "[parameters('app-value1')]",
+    "value2": "[parameters('app-value2')]",
+    "cpuLimit": "[parameters('app-cpuLimit')]",
+    "memoryLimit[0]": "[parameters('app-memoryLimit0')]",
+    "memoryLimit[1]": "[parameters('app-memoryLimit1')]"
+},
+```
+
+## UI Definition
+
+The following code snippet shows how the inputs can be defined in the UI definition:
+
+```json
+{
+    "name": "cpuLimit",
+    "type": "Microsoft.Common.TextBox",
+    "label": "CPU Limit",
+    "toolTip": "This will replace the cpuLimit's value inside your values.yaml",
+    "defaultValue": "{0.5, 1.0}",
```

Comment:
This doesn't seem to behave reliably. I have seen they got ' ' and end up getting into helm chart as string. I will remove this part of the solution